bibliofil_lists.admin.inc

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/**
+ * @file
+ * Module's administration functionality.
+ */
+
+/**
+ * Settings form.
+ */
+function bibliofil_lists_admin_settings_form($form, &$form_state) {
+  $form['bibliofil_lists_opac_url'] = [
+    '#type' => 'textfield',
+    '#title' => t('OPAC URL'),
+    '#description' => t("URL of client's OPAC service. Example: <i>https://presentasjon.bib.no</i>"),
+    '#default_value' => variable_get('bibliofil_lists_opac_url', ''),
+  ];
+
+  $form['#submit'][] = 'bibliofil_lists_admin_settings_form_submit';
+
+  return system_settings_form($form);
+}
+
+/**
+ * Custom form submit.
+ */
+function bibliofil_lists_admin_settings_form_submit($form, &$form_state) {
```

Comment:
Please add a validation handler as well which would validate the url.

Reply:
+